nss-day-cohort-45/nutyshell-running-with-scissors-1
# Nd task html/list

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
1) make sure both port and json are served
2) login, check to see rendering of tasks
3) logout and then back in with different user to see new list of tasks

# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
